content/how-to/cloud-servers/multifactor-authentication-from-the-myrackspace-portal/index.md

```diff
@@ -5,180 +5,171 @@ title: Multifactor authentication from the MyRackspace Portal
+
+**Important**: By default, we enable MFA account-wide. If you want to disable this setting, you need to generate a support ticket.
 
 ### Considerations
 
-- You cannot configure MFA from the Cloud Control Panel inside of the MyRackspace
-  Portal. You must log in to the
-  [Cloud Control Panel](https://login.rackspace.com) and use these
-  [instructions](/support/how-to/multifactor-authentication-from-the-cloud-control-panel/)
-  to set up your users for the Cloud Control Panel.
+- You must log in to the [MyRackspace Portal](https://my.rackspace.com) and use these
+  [instructions](/support/how-to/multifactor-authentication-from-the-my-rackspace-portal/)
+  to set up your users for the Rackspace Dedicated.
 
-- Administrators cannot set up devices for users. Users must configure their own
-  devices. To change MFA settings, users must log in to the MyRackspace Portal with
-  the user credentials they want to modify.
+- Administrators cannot set up devices for users. Users must configure their own devices. To  change MFA settings, users must log in to the MyRackspace Portal with the user credentials they want to modify.
 
-### Configure your account to authenticate by using an SMS device
+### Configure your account to authenticate by using SMS
 
-To configure your account to use an SMS device for MFA, you need the phone number
-associated with your digital device. Ensure that you have enabled the device to
-receive SMS text messages.
+To configure your account to use an SMS device for MFA, you need the phone number associated with your digital device. Ensure that you have enabled the device to receive SMS text messages.
 
-To register and verify an SMS device, use the following steps:
+To register and verify SMS, use the following steps:
 
 1. Log in to the [MyRackspace Portal](https://login.rackspace.com/).
 
-2. In the upper-right corner of the Cloud Control Panel, click the **user** menu
+2. In the upper-right corner of the MyRackspace Portal Panel, click the **user** menu
    and select **My Profile & Settings**.
 
-3. In the **Security Settings** section, select **Enable** for **Multi-factor
-   authentication**.
+<img width="947" alt="Dedicated 1" src="https://user-images.githubusercontent.com/98875667/161245007-03e8f041-943d-4062-8eba-80a4519a3955.png">
+
+3. In the **My Multi-Factor Devices** section, select **SMS**.
+
+<img width="517" alt="Dedicated 2" src="https://user-images.githubusercontent.com/98875667/161272990-bab8dc53-ef27-4a1b-b9d5-4ffe3f7a37a0.png">
 
-4. Select **User SMS** and click **Next**.
+4. Select **Add** and Select the country code for the device, type the device phone number and
+   click **Next**.
 
-5. Select the country code for the device, type the device phone number and
-   click **Next**
+<img width="444" alt="Dedicated 3" src="https://user-images.githubusercontent.com/98875667/161285146-61ba1cad-1bd6-4e80-a493-40c14116a50e.png">
```

Comment:
according to the previous documentation on this repo, it is preferred for the images to be located in the folder that this index.md is in. So within the `multifactor-authentication-from-the-myrackspace-portal` folder. Then update the link on this line.

Reply:
Done